### ISSUE TITLE
Organisation team promotion

### ISSUE THREAD

**Issue by lukegb** [SpongePowered/Ore#56]
@JBYoshi brings up the point that users might expect to have to add people to an organisation in order to add them to project teams

Project teams exist to help accommodate the case where users are using their personal namespace instead of creating organisations, and to add a little flexibility.

I'm planning on making the organisation team list page *include project teams* (which will actually just link to the project team manager) at the bottom of the list in the format `project name`/`team name`, an add a *Promote to Organisation Team* button on the project team management page. Does this seem sensible?

**Comment by JBYoshi** [comment]
Sounds good.

**Comment by gratimax** [comment]
I don't think that project teams really add any additional flexibility. Ignoring the existence of user projects, they're just organization teams that are restricted to a single project. So they're pretty much only necessary for projects that belong to users. What I'd really love is some sort of way to get rid of project teams entirely in favor of a generic 'teams' feature. I think that'd be much easier to use and understand instead of the somewhat complicated project/organization team hierarchy that there is now.
